### ISSUE TITLE
Restore no longer work, configure invalid repositories

### ISSUE THREAD

**Issue by TeddyAndrieux** [scality/metalk8s#2033]
**Component**:

'kubernetes'

**What happened**:

Since #1552 we no longer embbed all base and extras repositories in our archive. 
So repositories configuration from bootstrap script get removed but not from restore script.

```
[root@bootstrap-rhel ~]# /srv/scality/metalk8s-2.4.0-dev/restore.sh -b /var/lib/metalk8s/backup_20191017_133051.tar.gz -v
> Extract backup archive '/var/lib/metalk8s/backup_20191017_133051.tar.gz'...
> Pre-minion system tests...
> Disabling Salt minion service...
> Stopping Salt minion service...
> Configuring local YUM repositories...
find: ‘/srv/scality/metalk8s-2.4.0-dev/packages/redhat/metalk8s-base-el7’: No such file or directory
```

**What was expected**:

Not trying to configure `metalk8s-base` and `metalk8s-extra` when restoring.

**Steps to reproduce**

Run restore script

**Resolution proposal** (optional):

Change from this commit dfabb912c1a818856821d334f1e4c44c8e91a1d3 from bootstrap script should be done in restore script also.
Basically just remove `metalk8s-base`, `metalk8s-extras` and `metalk8s-updates` from `GPGCHECK_REPOSITORIES`.
